.kokoro/samples/lint/common.cfg

```diff
@@ -31,4 +31,17 @@ gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/python-docs-samples"
 gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 
 # Use the trampoline script to run in docker.
-build_file: "python-retail/.kokoro/trampoline_v2.sh"
\ No newline at end of file
+build_file: "python-retail/.kokoro/trampoline_v2.sh"
+
+env_vars: {
+    key: "PROJECT_NUMBER"
+    value: "945579214386"
+}
+env_vars: {
+    key: "BUCKET_NAME"
+    value: "crs-interactive-tutorials"
+}
+env_vars: {
+    key: "EVENTS_BUCKET_NAME"
+    value: "crs-interactive-tutorials"
+}
```

Comment:
From our meeting, we created the `retail-samples-secrets` secret so we can set the `SECRET_MANAGER_KEYS` env to `retail-samples-secrets`

Reply:
I have added the SECRET_MANAGER_KEYS=retail-samples-secrets to the presubmit/common.cfg